packages/components/src/components/files/FileAttachmentForm.tsx

```diff
@@ -382,7 +410,7 @@ export class FileAttachmentForm extends React.Component<FileAttachmentFormProps,
                     />
                 )}
                 {this.renderFooter()}
-                {showButtons && this.renderButtons()}
+                {!compact && showButtons && this.renderButtons()}
```

Comment:
Do you know if/where in the apps or LKS there are usages of the FileAttachmentForm with compact set to true? Just curious where that might be and if those would be affected at all by these changes.

Reply:
I created the compact buttons for the MultiAuth story, and I looked through instances of <FileAttachmentForm with compact set to true in my trunk (which includes sampleManagement and Biologics), and ui-components. I haven't seen it used elsewhere so far, but I'll ask FrontEnd if anyone had used it, as well!

Reply:
Yeah, you could ask, but it sounds like you would have caught any usages by looking in platform, SM, Biologics, and ui-components. I had forgotten that compact was added for MultiAuth so I assume that is the only current usage, so you should be good if you checked that one. Thanks.